venia-integration-tests/src/tests/pageBuilder/homePage.spec.js

```diff
@@ -0,0 +1,13 @@
+/// <reference types="cypress" />
+
+context('Assert Venia Home Page pagebuilder content', () => {
+    describe('Assert Page Builder content', () => {
+        it('should render page builder content correctly', () => {
+            cy.visit('/');
+
+            cy.wait(3000);
```

Comment:
I feel like this is too short. A no-cache, hard refresh page load on https://develop.pwa-venia.com/ took 6000ms for me.

Maybe 10000 here?

Reply:
https://docs.cypress.io/faq/questions/using-cypress-faq#How-can-I-make-Cypress-wait-until-something-is-visible-in-the-DOM

Reply:
Yeah, you guys are right, but in the scope of this PR, I was working on introducing the plugin and integrating it not dint quite get into the test itself. In the following tickets, we can delegate this to the developer working on the ticket about what they think is the right choice of wait or use a helper as Dave suggested.

The test in this PR is a dummy one, just there to prove that this plugin works after the integration.